.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "etc/grafana/provisioning/plugins/groupedbarchart-panel"]
+	path = etc/grafana/provisioning/plugins/groupedbarchart-panel
+	url = https://github.com/gipong/grafana-groupedbarchart-panel.git
```

Comment:
Git submodule is evil :smiling_imp: I'd prefer to `curl` the project's master state [1] in a `plugins` Makefile rule WDYT?

[1] https://github.com/gipong/grafana-groupedbarchart-panel/archive/refs/heads/master.zip

Reply:
Great Idea) Thanks) Updated the Makefile)